4.1/init.sh

```diff
@@ -66,7 +70,7 @@ sudo -E -u postgres psql postgres -tAc "SELECT 1 FROM pg_roles WHERE rolname='ww
 sudo -E -u postgres psql postgres -tAc "ALTER USER nominatim WITH ENCRYPTED PASSWORD '$NOMINATIM_PASSWORD'" && \
 sudo -E -u postgres psql postgres -tAc "ALTER USER \"www-data\" WITH ENCRYPTED PASSWORD '${NOMINATIM_PASSWORD}'" && \
 
-sudo -E -u postgres psql postgres -c "DROP DATABASE IF EXISTS nominatim"
+sudo -E -u postgres psql postgres -c "DROP DATABASE IF EXISTS $PGDATABASE"
```

Comment:
I'm wondering why we even drop the database in the first place. Seems quite destructive.

@philipkozeny do you have an idea?

Reply:
@leonardehrenfried because the OG maintainer started doing that in the initial version he released, never questioned if we should continue with that behaviour to be honest ... https://github.com/mediagis/nominatim-docker/blob/10fc9e44f2eed5d3d33cb36c65ee9b930b63e143/2.5/Dockerfile#L50

Reply:
That because I try to make it safely.
It's OK, if you could change name, I think